salt/modules/mandrill.py

```diff
@@ -238,7 +238,7 @@ def send(message,
     data = {
         'key': params['api_key'],
         'message': message,
-        'async': async,
+        'asynchronous': asynchronous,
```

Comment:
Should the dict here still be 'async'?

Reply:
Oh, you're right! And I didn't deprecated it properly as well.